css/main.css

```diff
@@ -1,19 +1,439 @@
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
 body {
-  font-family: 'Lato', sans-serif;
+  margin: 0;
+  font-family: 'Lato';
+  text-align: justify;
```

Comment:
Bad idea. Please specify text-align: justify only for separate components. There is no need to set this property globally.

Reply:
thanks for the comments, I will correct all mistakes!